src/components/cssComponents/Toast.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import App from "./../../App";
+
+function Toast() {
+  const [toastStyle, setToastStyle] = useState({ display: "none" });
+  return (
+    <App
+      header="Toast/Snackbar"
+      description="You can use this toast for the purpose of small timed notification"
+    >
+      <div className="content-center">
+        <button
+          className="toast-btn btn-secondary mb-4"
+          onClick={() => {
+            setToastStyle({ display: "block" });
+            setTimeout(() => {
+              setToastStyle({ display: "none" });
+            }, 3000);
+          }}
+        >
+          Toast/SnackBar Button
+        </button>
+      </div>
+      <div className="toast dark text-white" style={toastStyle}>

```

Comment:
Toast message generally appears at the bottom-right, bottom-left, and top-right of the page.